### ISSUE TITLE
JavaScript error in Designer: getElementById()

### ISSUE THREAD

**Issue by ibennetch** [phpmyadmin/phpmyadmin#14079]
In current master, when I am in Designer and click on the gear icon for a row (the gear icon at the top for each table works as expected).

Web Console has this error text:
```
TypeError: document.getElementById(...) is null
[Learn More]
move.js:1732:26
Click_option
http://192.168.56.2/pma-dev/fork/js/designer/move.js:1732:26
<anonymous>
http://192.168.56.2/pma-dev/fork/js/designer/move.js:2091:9
new_func
http://192.168.56.2/pma-dev/fork/js/error_report.js:248:28
dispatch
http://192.168.56.2/pma-dev/fork/js/vendor/jquery/jquery.min.js:2:41720
$event.dispatch
http://192.168.56.2/pma-dev/fork/js/vendor/jquery/jquery.event.drag-2.2.js:374:9
add/y.handle
http://192.168.56.2/pma-dev/fork/js/vendor/jquery/jquery.min.js:2:39774
```

**Comment by mauriciofauth** [comment]
@ibennetch This button should not always be visible and I have already fixed this in commit ca7fd2b2c737c997060fbb67947d9cb3d9aa7b09, but I don't know in what cases it should appear to test whether this issue has been fixed or not.

**Comment by mauriciofauth** [comment]
These buttons should only be visible when you are in Visual Builder mode.
For example, when you click on _Switch to visual builder_ on QBE page: https://demo.phpmyadmin.net/master-config/db_qbe.php?server=1&db=sakila

So, apparently the commit ca7fd2b2c737c997060fbb67947d9cb3d9aa7b09 has fixed the bug, so I'll close this issue.
